tests/fast-integration/skr-test/test.js

```diff
@@ -56,18 +56,16 @@ describe('SKR test', function() {
     await initK8sConfig(shoot);
   });
 
-  it('Execute the tests', async function() {
```

Comment:
This `it` block didn't wait until the two test run successfully. Instead it instantly reported `✓` even though the test were still running. Removing the it block solves it